InternshipBE/DAL/Repositories/BestContactTimeRepository.cs

```diff
@@ -0,0 +1,22 @@
+using DAL.Database;
+using DAL.Entities;
+using DAL.Interfaces;
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DAL.Repositories
+{
+	public class BestContactTimeRepository : GenericRepository<BestContactTime>, IBestContactTimeRepository
+	{
+		public BestContactTimeRepository(ApplicationDbContext context) : base(context)
+		{
+		}
+
+		public async Task<List<BestContactTime>> GetAllByUserIdAsync(string userId)
+		{
+			return await _context.BestContactTimes.Where(x => x.User.Id == userId).ToListAsync();
```

Comment:
Include

Reply:
you can add UserId filed to BestContactTime
![image](https://user-images.githubusercontent.com/71830071/140882116-d363489f-f37a-44d4-965f-77380fe0fff3.png)